tests/integration/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ if(CORENRN_ENABLE_GPU)
   set(GPU_ARGS "--nwarp 8 --gpu")
 endif()
 
-# List of tests with arguments
-set(TEST_CASE_ARGS
+# List of tests with their arguments
+set(TEST_CASES
```

Comment:
@alkino : FYI, in CLI11 PR you renamed these but forgot to rename at line 68. So none of integrations are running after CLI11 merge.